assets/_locales/en/messages.json

```diff
@@ -1,4 +1,8 @@
 {
+  "fallback": {
```

Comment:
Can you please also add this to the other languages. Just the same value for now (English), someone will go throw all of them before each release.